src/cmd/linuxkit/run_vmware.go

```diff
@@ -68,9 +74,10 @@ func runVMware(args []string) {
 		fmt.Printf("Options:\n")
 		vmwareArgs.PrintDefaults()
 	}
-	runCPUs := vmwareArgs.Int("cpus", 1, "Number of CPUs")
-	runMem := vmwareArgs.Int("mem", 1024, "Amount of memory in MB")
-	runDisk := vmwareArgs.String("disk", "", "Path to disk image to use")
+	vmwareCPUs := vmwareArgs.Int("cpus", 1, "Number of CPUs")
+	vmwareMem := vmwareArgs.Int("mem", 1024, "Amount of memory in MB")
+	vmwareDisk := vmwareArgs.String("disk", "", "Path to disk image to use (.vmdk appended if disk is created)")
```

Comment:
Thought it was easier for the user, as we don't specify extensions for .iso or .vmdk. I suppose explicit paths will make it easier for a user to test something by pulling in an existing disk from elsewhere on a system.